pkg/cache/keyfunc.go

```diff
@@ -36,36 +79,37 @@ func DeletionHandlingMetaClusterNamespaceKeyFunc(obj interface{}) (string, error
 	return MetaClusterNamespaceKeyFunc(obj)
 }
 
+// ObjMetaClusterNamespaceKey is a convenient default KeyFunc which knows how to make
+// structured keys for API objects which implement meta.Interface.
+// This is the structured alternative to MetaClusterNamespaceKeyFunc;
+// putting such an object reference in a queue means that no parsing errors are possible downstream.
+func ObjMetaClusterNamespaceKey(obj interface{}) (Key, error) {
```

Comment:
nit: the convention is to call this a `KeyFunc`

Reply:
I agree! I only point it out since it might make it clearer for consumers that are used to that convention.